src/main/java/Pair.java

```diff
@@ -0,0 +1,67 @@
+public class Pair {
```

Comment:
could use a better name since it is not a generic pair.

Reply:
Oops, It was in my mind, but didn't changed it later.